starboard/shared/posix/socket_accept.cc

```diff
@@ -36,13 +36,15 @@ SbSocket SbSocketAccept(SbSocket socket) {
 
   int socket_fd = HANDLE_EINTR(accept(socket->socket_fd, NULL, NULL));
   if (socket_fd < 0) {
+    errno = SbSystemGetLastError();
```

Comment:
Do we need to set errno for the functions in `starboard/shared/posix`? I would expect that those don't need any changes.